dts/bindings/sensor/adi,adxl345-spi.yaml

```diff
@@ -5,4 +5,4 @@ description: ADXL345 3-axis accelerometer with SPI connection
 
 compatible: "adi,adxl345"
 
-include: [sensor-device.yaml, spi-device.yaml]
+include: [sensor-device.yaml, spi-device.yaml, "adi,adxl345-common.yaml"]
```

Comment:
You can remove `sensor-device.yaml` here since it's now being included via `adi,adxl345-common.yaml`